deepinv/tests/test_physics.py

```diff
@@ -1317,3 +1317,40 @@ def test_adjoint_autograd(name, device):
     delta_y = y.grad
     Az = physics.A(z)
     assert torch.allclose(delta_y, Az, rtol=1e-5)
+
+
+@pytest.mark.parametrize("name", OPERATORS)
+def test_clone(name, device):
+    physics, imsize, _, dtype = find_operator(name, device)
+
+    physics_clone = physics.clone()
```

Comment:
also the simplest `assert physics.__name__ == physics_clone.__name__` or `isinstance(physics_clone, physics.__class__)`